AfsprakenRegels.md

```diff
@@ -461,31 +461,53 @@ aanduiding gedefinieerd worden.
+- de gegevens over de relatie bestaan alleen zolang de relatie tussen beide objecten bestaat en zolang elk van beide objecten zelf (nog) bestaan. wanneer een van beide niet meer bestaat, bestaat de relatie niet meer en hiermee komen ook de gegevens over de relatie te vervallen. 
+
+_Uitzondering situatie - dezelfde gegevens voor meerdere relaties_
+
 Het is echter mogelijk dat dezelfde gegevens voor meerdere relaties tegelijk
-gelden. Het is dan niet mogelijk om het te modelleren als relatieklasse. Wel
-gewenst, maar het kan niet als UML-associationClass. Als deze uitzondering het
-geval is, dan worden de relatieklasses gemodelleerd als «Objecttype», met één
-inkomende relatie en één uitgaande relatie. De oorspronkelijke kardinaliteit van
-de beoogde relatieklasse wordt hierbij behouden.
+gelden. 
+
+Het is dan niet mogelijk om het te modelleren als relatieklasse. Wel gewenst, 
+maar het kan niet als UML-associationClass. 
+Als deze uitzondering het geval is, dan worden de relatieklasses gemodelleerd als «Objecttype», 
+met één inkomende relatie en één uitgaande relatie. 
+De oorspronkelijke kardinaliteit van de beoogde relatieklasse wordt hierbij behouden. 
+
+De relatie is geobjectiveerd. Elke betrokken relatie wordt in dit geval behandeld als een object met gegevens. 
 
 <pre class='example'>
 Een Perceel kan vanwege een Perceel splitsing overgaan in twee of meerdere
-andere Percelen. De ‘overgegaan in’ relatie wordt bijgehouden in een
-relatieklasse. Gegevens over de splitsing zijn voor al deze relaties gelijk.
+andere Percelen. De ‘overgegaan in’ relatie wordt bijgehouden als een
+relatieklasse met eigen (splitsing) gegevens. Bij een splitsing ontstaan * nieuwe percelen 
+en er is daarom sprake van meerdere relaties. De gegevens over de splitsing zijn echter altijd 
+voor al deze relaties gelijk. Om deze reden wordt de relatieklasse 'overgegaan in' gemodelleerd als een objecttype.  
 </pre>
 
-Het metamodel ondersteunt (nog) geen relatieklassen tussen drie of meer
-objecttypen. Dit kan in uw eigen extensie toegevoegd worden.
-
 <pre class='ex-generic'>
 Een CONTRACT kan bijvoorbeeld ook een afspraak zijn tussen twee
 óf méér SUBJECTen, waarbij de gegevens van de relatie voor alle betrokken
 objecten hetzelfde zijn. CONTRACT wordt dan gemodelleerd als objecttype, waarbij
-beschreven wordt wat er moet gebeuren wanneer één van de SUBJECTen niet meer
-bestaat.
+beschreven wordt wat er moet gebeuren wanneer één van de SUBJECTen niet meer bestaat.
 </pre>
 
+Opmerkingen: 
+- de gegevens over de relatie bestaan alleen zolang de relatie tussen beide objecten bestaat en zolang elk van beide objecten zelf (nog) bestaan. Dit is echter niet meer te zien aan de metaklasse, omdat dit «Objecttype» is geworden en niet meer «Relatieklasse» is. Geef daarom in de toelichting van het objecttype aan dat elk object van dit objecttype bestaansafhankelijk is van de beide objecttypen die verbonden zijn door deze geobjectiveerde relatie.  
```

Comment:
> [...] en zolang elk van beide objecten zelf (nog) bestaan.

Ik denk niet dat het nodig is om het te hebben over het bestaan van de objecten. Het is voldoende om het te hebben over of de relatie nog bestaat, gezien de relatie het onderwerp is van de gegevens van de instantie van de bijbehorende relatieklasse.

> Dit is echter niet meer te zien aan de metaklasse, omdat dit «Objecttype» is geworden en niet meer «Relatieklasse» is. Geef daarom in de toelichting van het objecttype aan dat elk object van dit objecttype bestaansafhankelijk is van de beide objecttypen die verbonden zijn door deze geobjectiveerde relatie.

Ik snap niet echt wanneer dit optreedt. Behalve misschien in een vrij technische situatie in een database of zo. Maar hebben we het daarover?